tests/template_test.py

```diff
@@ -0,0 +1,44 @@
+# fmt: off
```

Comment:
I would recommend to entirely move this file away from the `tests` folder: 

* no chance to mess up with real tests
* you can easily put whatever you want inside it

Reply:
Done